### ISSUE TITLE
Does not compile

### ISSUE THREAD

**Issue by darxkies** [docker/go-plugins-helpers#74]
Hi 

I am getting the following error.

github.com/docker/go-plugins-helpers/sdk/unix_listener.go:20: cannot use gid (type int) as type string in argument to sockets.NewUnixSocket

That is due to the fact that I stick to the last tagged version of docker/go-connections (v0.2.1).

go-plugins-helpers seems to use the master version of docker/go-connections.

Wouldn't it be better to also use only tagged versions of go-connections?

Thank you.

**Comment by dave-tucker** [comment]
@darxkies thanks for the report.
IIRC, the Golang best practice was for libraries not to specify versions of their dependencies but instead for that to be the responsibility of the consumers. I don't think this has changed, and I think this remains a pretty sensible default but I'm open to suggestions.
As a workaround, bumping your version of `go-connection` should fix the issue.

**Comment by darxkies** [comment]
You are right. I will hack the appropriate commit id into the glide config to avoid the error.

Thank you.
